### PR TITLE
Add changelog & translate contributing.md

### PR DESCRIPTION
**Pull request by DareJare**

### CODE REVIEW

**Comment by hjk41** [comment]
I think the English version of the docs should be put in /doc

**Comment by knightast** [comment]
the doc directory should be:
doc/zh-CN/...
doc/en-US/...

Please put the doc in the right directory, @KingSkyLi  will do the rest of English translation in one month.

File: README.md [hunk 1]
```diff
@@ -66,6 +66,43 @@ for example, `tugraph/tugraph-env-centos7:1.1.0`, which can help developers get
 
 For more details, please refer to the docs in [doc-zh]
 
+## 4. Change Log
```
Comment (knightast) [inline README.md:69]: I don't think the chang log of pr is necessary in this README, which only contains very high-level introduction.

**Comment by knightast** [comment]
emm... let me make it more clear.

Now we have:
doc/FILE_EN.md
doc-zh/FILE_CH.md

We want it to be:
doc/en-US/FILE.md
doc/zh-CN/FILE.md

Or I can have a pr on your branch.

**Comment by knightast** [comment]
A pr has merged to master to put the doc in the right directory, you can update this file name now.

**Pull request closed without merging**